dr_ensenso/CMakeLists.txt

```diff
@@ -15,6 +15,13 @@ find_package(Eigen REQUIRED)
 find_package(OpenCV REQUIRED)
 find_package(PCL REQUIRED)
 
+find_package(OpenMP REQUIRED)
+if(OPENMP_FOUND)
```

Comment:
It's pointless to check `OPENMP_FOUND` if you already pass `REQUIRED` to `find_package`. It could be useful if you want to make openmp support optional. But if not, the check should just go.